sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/filesystem/FileSystemLogger.java

```diff
@@ -56,9 +56,9 @@ void doLog(Logger logger) {
 
   private void logEncoding(Logger logger, Charset charset) {
     if (!fs.isDefaultJvmEncoding()) {
-      logger.info("Source encoding: " + charset.displayName() + ", default locale: " + Locale.getDefault());
+      logger.info("Source encoding: {}, default locale: {}", charset.displayName(), Locale.getDefault());
```

Comment:
a Supplier can be used to avoid a call to displayName() and Locale.getDefault() if INFO level is disabled.